helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -163,6 +164,8 @@ processors:
       - action: insert
         key: receiver
         value: k8scluster
+      - action: delete
+        key: k8s.pod.start_time
```

Comment:
This one I'm not sure about. It's needed to make tests work correctly, but line 523 in functional_test.go should already fix that, so there is something weird going on somewhere. @VihasMakwana